pkg/protocol/client/client.go

```diff
@@ -220,7 +220,8 @@ func doRequestFollowRedirectsBuffer(ctx context.Context, req *protocol.Request,
 
 	statusCode, _, err = DoRequestFollowRedirects(ctx, req, resp, url, defaultMaxRedirectsCount, c)
 
-	body = bodyBuf.B
+	// In HTTP2 scenario, client use stream mode to create a request and its body is in body stream.
```

Comment:
Why stress “HTTP2 scenario”? If it is about body stream, HTTP/1.1 also has the kind of issue?

Reply:
HTTP2 is a scenario that must trigger this bug. And in HTTP1, only client recv body exceed max size and client is in stream mode can trig it.